app/models/katello/content_view_version.rb

```diff
@@ -66,6 +66,8 @@ class ContentViewVersion < Katello::Model
       joins(:repositories).includes(:content_view).merge(repositories).distinct
     end
 
+    scope :latest, -> { order('major DESC').order('minor DESC') }
```

Comment:
Since this returns a list of versions, maybe `by_latest` or `by_version_number` is a better name?

Reply:
https://guides.rubyonrails.org/active_record_querying.html#ordering says you can use a single order statement